.github/workflows/nightly/forward-postgres/action.yaml

```diff
@@ -24,6 +24,11 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Build forward-postgres-tester
+      run: |
+        $(cd cmd/nightly/forward-postgres-tester/ && make build)
+      shell: bash
+
```

Comment:
Maybe pre-optimalization, but it would be possible to add a cache action to avoid building every time.

Reply:
Definitely possible. However this small program takes ~10 seconds to build. I'll save it for when we need to optimize yes